static/js/src/publisher-admin/routes/root.tsx

```diff
@@ -22,17 +22,9 @@ function Root() {
   return (
     <>
       <SectionHeader />
-      <div className="l-application">
-        <div className="l-main">
-          <div className="p-panel">
-            <div className="p-panel__content">
-              <div className="u-fixed-width">
-                <SectionNav />
-                <Outlet />
-              </div>
-            </div>
-          </div>
-        </div>
+      <div className="u-fixed-width">
```

Comment:
So this is a root of all the views? Not just the Publicise view?

Could this be a case that application layout is used for something somewhere (on other views)?
It is just the Listing/Releases/Publicise/etc tabs, or may it be reused elsewhere as well?

Seems like we would need to QA more than just the Publicise page to make sure there are no regressions.